chowpronto_frontend/src/shared_components/CloseIcon.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+function Icon() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      // width="25"
+      // height="25"
```

Comment:
We need to start tidying up our code and remove all comments

Reply:
will create a new PR for this